### ISSUE TITLE
Confirm a11y for accordion in simple menu

### ISSUE THREAD

**Issue by adneforslund** [equinor/energyvision#801]

**Comment by fernandolucchesi** [comment]
Probably duplicate of #895

**Comment by wenche** [comment]
Since it's mainly used for one menu item with multiple sub items within a list of single items, it's hard to do this in another way. I think this should be good enough since accordion is nothing but a bunch of divs and aria states and not a "real" html element. The menu as a whole should be tested with focus on a11y at some point to see if we can improve it.
